src/csvtsinputfileconn.cc

```diff
@@ -515,18 +515,30 @@ namespace dd
           out << " " << _label_pos[i] << " " << delim;
         out << " " << _label_pos[_label_pos.size() - 1] << std::endl;
       }
-    out << "min_vals: ";
-    for (unsigned int i = 0; i < _min_vals.size() - 1; ++i)
-      out << " " << std::setprecision(boundsprecision) << _min_vals[i] << " "
-          << delim;
-    out << " " << std::setprecision(boundsprecision)
-        << _min_vals[_min_vals.size() - 1] << std::endl;
-    out << "max_vals: ";
-    for (unsigned int i = 0; i < _max_vals.size() - 1; ++i)
-      out << " " << std::setprecision(boundsprecision) << _max_vals[i] << " "
-          << delim;
-    out << " " << std::setprecision(boundsprecision)
-        << _max_vals[_max_vals.size() - 1] << std::endl;
+    if (_min_vals.size() > 0)
+      {
+        out << "min_vals: ";
+        for (unsigned int i = 0; i < _min_vals.size() - 1; ++i)
+          out << " " << std::setprecision(boundsprecision) << _min_vals[i]
+              << " " << delim;
+        out << " " << std::setprecision(boundsprecision)
+            << _min_vals[_min_vals.size() - 1] << std::endl;
+      }
+    else
```

Comment:
`min_vals` and `max_vals` only apply when `scale` is set to true. Does this code raise an exception if `scale` is set to false ?

Reply:
`serialize_bounds` is never called when `scale` is set to true ([here](https://github.com/jolibrain/deepdetect/blob/665b0ba2a921375bdef24ff53e7a194d88b8b84b/src/csvtsinputfileconn.cc#L164))

Reply:
OK good, I guess you mean when scale is set to false.